config/prod.exs

```diff
@@ -12,17 +12,19 @@ use Mix.Config
 # manifest is generated by the mix phoenix.digest task
 # which you typically run after static files are built.
 config :remote_retro, RemoteRetro.Endpoint,
+  server: true,
   http: [port: {:system, "PORT"}],
-  url: [scheme: "https", host: System.get_env("HEROKU_HOST"), port: 443],
+  url: [scheme: "https", host: "bar.gigalixirapp.com", port: 443],
   force_ssl: [rewrite_on: [:x_forwarded_proto]],
   cache_static_manifest: "priv/static/manifest.json",
-  secret_key_base: System.get_env("SECRET_KEY_BASE")
+  secret_key_base: "${SECRET_KEY_BASE}"

```

Comment:
@jesseshieh curious if the shift to `"${ENV_VAR_NAME}"` is a syntactic preference? the `System.get_env` calls seem to work just fine, and are less magical (to my eyes).

Reply:
`System.get_env` is evaluated at compile time. Since your env vars are available at compile time, often this works fine. But it means anytime you change an env var, you'll need to recompile. `${FOO}` is distillery syntax which allows you to actually replace the value at runtime which is more what people expect.

Reply:
woah. We'll go the distillery route 😄